data/wanderers start.txt

```diff
@@ -376,16 +376,59 @@ mission "Tell Wanderers about the Pug"
 				decline
 
 
+mission "Wanderers: Defend Vara Ke'sok Hint"
+	name "Wanderer supplies to <planet>"
+	description "Transport <cargo> to <system>, where repairs are needed from Unfettered attacks."
+	passengers 1
+	cargo "building supplies" 20
+	source 
+		near "Sich'ka'ara" 1 100
+	destination "Vara Ke'sok"
+	to offer
+		random < 60
+		has "Wanderers Conversation: offered"
+		not "Wanderers: Defend Vara Ke'sok: done"
+	on offer
+		conversation
+			`Walking around the spaceport, you are approached by two Wanderers. One of them begins speaking in the Hai language, which your translator picks up. "Greetings, [stranger, curiosity]. We heard you perform [charitable, heroic] acts. Is this correct?"`
```

Comment:
Where'd they hear that from? 🤔

Reply:
Added a section fleshing out where they heard it from.